### PR TITLE
[location-search] JSRS migration VCAP-24705

### PR DESCRIPTION
**Pull request by gabrieloliveira41**

### CODE REVIEW

**Comment by gabrieloliveira41** [comment]
the migration tool successfully migrated the code, theres only one deprecation on the code
image
but if i use the recomended version of viv.geo the capsule presented diferent results from the version before the migration

**Comment by gabrieloliveira41** [comment]
image
also California is spelled wrong on the trainings section

**Comment by gabrieloliveira41** [comment]
I talked with viv.geo owner from PT-BR and there's no viable way to fix that deprecation now without changing a lot of the current capsule

**Comment by gabrieloliveira41** [comment]
Hello @kangawoo44 @abapayeViv, could you review my PR, please?

File: location-search/capsule.bxb [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
+      version-from (location)
+    }
   }
   permissions {
     device-location-access
     library-permission (contact:contacts)
     library-permission (self:profile)
   }
-  runtime-version (8)
+  runtime-version (8) {
```
Comment (abapayeViv) [inline location-search/capsule.bxb:31]: We should update the runtime-version to the latest stable one listed on our [dev docs](https://corp.bixbydevelopers.com/dev/docs/reference/type/capsule.runtime-version) any time we update moving forward.

Can you update this runtime version to runtime-version (9)?

**Comment by kangawoo44** [comment]
I will get back on this as well since the warning's about library capsule deprecations.

**Comment by gabrieloliveira41** [comment]
the same warning message is shown in the capsule before the migration
image

**Review by kangawoo44 — APPROVED** [review]

**Comment by kangawoo44** [comment]
since these warnings were there before the JSRS migration we'll merge this change and work on fixing the library warnings separately.

**Comment by gabrieloliveira41** [comment]
Hello @abapayeViv, could you merge this PR?

**Pull request merged**